contracts/AcrossConfigStore.sol

```diff
@@ -8,7 +8,8 @@ import "@openzeppelin/contracts/access/Ownable.sol";
  * @title Allows admin to set and update configuration settings for full contract system. These settings are designed
  * to be consumed by off-chain bots, rather than by other contracts.
  * @dev This contract should not perform any validation on the setting values and should be owned by the governance
- * system of the full contract suite..
+ * system of the full contract suite.
+ * @custom:security-contact bugs@umaproject.org
```

Comment:
nit: should we change this to bugs@across.to? (We can change on the docs site too)